docs/Versioning.md

```diff
@@ -25,36 +33,33 @@ For the most up-to-date builds, the following maven repo should be added to the
 </repositories>
 ```
 
-Then the following dependency can be added:
-
-```
-  <groupId>org.foundationdb</groupId>
-  <artifactId>fdb-record-layer-core</artifactId>
-  <version>4.0.+</version>
-```
-
-Older builds are also available in the maven central repository, and newer builds will be published there upon the resolution of [Issue #1288](https://github.com/FoundationDB/fdb-record-layer/issues/1288).
+See: [Issue #1288](https://github.com/FoundationDB/fdb-record-layer/issues/1288).
 
 ### Gradle Dependency
 
-For the most up-to-date builds, the following maven repo should be added to the repositories section:
+For the most up-to-date builds, the following dependency can be added to gradle:
 
 ```
 repositories {
   mavenCentral()
-  maven {
-    url "https://ossartifacts.jfrog.io/artifactory/fdb-record-layer/"
-  }
+}
+
+dependencies {
+  implementation "org.foundationdb:fdb-record-layer-core:4.1.+"
 }
 ```
 
-Then the following dependency can be added:
+For builds older than 4.1 but newer than 2.8, an additional repository must be added to the build configuration, as older versions were not published to Maven Central:
 
 ```
-"org.foundationdb:fdb-record-layer-core:4.0.+"
+repositories {
+  maven {
```

Comment:
Should this have `mavenCentral()` ?
```suggestion
  mavenCentral()
  maven {
```

Reply:
I actually don't think so. You only need `mavenCentral()` for our dependencies, and it does note above that you need an _additional_ repository (beyond maven central) to get the RL build here